core/accounts.go

```diff
@@ -141,7 +141,7 @@ func createTokenWithOauth2(config *tomlConfig, user string, url string) (string,
 
 	token := KongOauth2Token{}
 	ko := &KongConsumerOauth2{
-		Name:         EdgeXService,
+		Name:         config.SecretService.SNIS,
```

Comment:
As I pointed out in #37 , the `Name` parameter here should be an independent property in configuration.toml that is separate from everything else so that different consumers can be registered with Kong for the same SNIS.